modules/shapefile/src/lib/parsers/parse-dbf.js

```diff
@@ -251,3 +264,29 @@ function parseNumber(text) {
 function parseCharacter(text) {
   return text.trim() || null;
 }
+
+/**
+ * Create a standard Arrow-style `Field` from field descriptor.
+ * TODO - use `fieldLength` and `decimal` to generate smaller types?
+ * TODO - is everything nullable. Can we set this false for some fields?
```

Comment:
If I recall correctly that isn't in the DBF's metadata, so every column needs to be considered nullable.